x/wasm/internal/types/types.proto

```diff
@@ -32,6 +32,7 @@ message Params {
     option (gogoproto.goproto_stringer) = false;
     AccessConfig code_upload_access = 1 [(gogoproto.nullable) = false, (gogoproto.moretags) = "yaml:\"code_upload_access\""];
     AccessType instantiate_default_permission = 2 [(gogoproto.moretags) = "yaml:\"instantiate_default_permission\""];
+    uint64 max_wasm_code_size = 3 [(gogoproto.moretags) = "yaml:\"max_wasm_code_size\""];
```

Comment:
Yup. Nice addition here.

Reading the config option (1k gas) on upload (> 1 million gas) is virtually 0 overhead. Happy for a param here.